src/qt/overviewpage.h

```diff
@@ -49,6 +50,7 @@ public Q_SLOTS:
     ClientModel *clientModel;
     WalletModel *walletModel;
     interfaces::WalletBalances m_balances;
+    bool m_privacy{false};
```

Comment:
I think this should be in `OptionsModel`. No need to go through bitcoingui -> walletframe -> walletview -> overview.

Reply:
> I think this should be in `OptionsModel`.

In this PR the privacy mode state is not intended to be preserved between consequent `bitcoin-qt` runs, that an `OptionsModel` instance does.

I'd leave the detailed discussion for a followup pull.

Reply:
It doesn't have to be persisted to be in `OptionsModel`, it's a runtime option only for now.

Reply:
Sorry, going to leave it as is.